hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/Service.java

```diff
@@ -20,18 +20,10 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 
 /**
- * A definition of an interface that will be implemented by each conceptual "service" like
- * crypto-service, token-service etc.,
+ * A definition of an interface that will be implemented by each conceptual "service" like crypto-service, token-service
+ * etc.,
  */
 public interface Service {
-    /**
-     * Returns the name of the service. This name must be unique for each service deployed on the
-     * application.
-     *
-     * @return the name
-     */
-    @NonNull
-    String getServiceName();
```

Comment:
I don't think it makes much sense to remove this method right now. We may have to put it back quite soon or replace it with very similar functionality.